---
title: Session 3 (29nd April 2020)
date: "2020-04-29T16:00:00Z"
layout: post
draft: false
path: "/posts/session-3"
category: "Session Schedule"
tags:
  - "Clinical microbiology"
  - "Population genetics and Microbial epidemiology"
description: "With talks from Conor Meehan, Leonardo de Oliveira Martins and Rebecca Bengtsson"
---
# Schedule 

Time: 16:00 29 April 2020 (UTC) 

Zoom link: https://cdc.zoomgov.com/j/1613270485

Discord link: https://discord.gg/N9dVFJp

Add to your calendar: [.iCal file](./Micro_Binfie_Virtual_conference.ics)

* **16:00 Discovery of a new lineage of *Mycobacterium tuberculosis*** 
  Conor Meehan (University of Bradford) Video: https://youtu.be/5x8B4J6hFFU
* **16:30 From phylogenetics to phylogenomics: between bacterial strains and the tree of life**  
  Leonardo de Oliveira Martins  (Quadram Institute Bioscience) Video: https://youtu.be/N0fkRuhr0RE
* **17:00 Combating *Shigellosis* in endemic settings with pathogen genomics**  
  Rebecca Bengtsson (The University of Liverpool) Video: https://youtu.be/M6qjwWG-u0o
* **17:20 Virtual coffee break**

## Discovery of a new lineage of *Mycobacterium tuberculosis*
The human- and animal-adapted lineages of the *Mycobacterium tuberculosis* complex (MTBC) are thought to have clonally expanded from a common progenitor in Africa. However, the molecular events that accompanied this emergence remain largely unknown. Here, we describe two MTBC strains isolated from patients with multidrug-resistant tuberculosis, representing an as-yet-unknown lineage, named Lineage 8 (L8), restricted to the African Great Lakes region. Using genome-based phylogenetic reconstruction, we show that L8 is a sister clade to the known MTBC lineages. Comparison with other complete mycobacterial genomes indicate that the divergence of L8 preceded the loss of the cobF genome region - involved in the cobalamin/vitamin B12 synthesis - and gene interruptions in a subsequent common ancestor shared by all other known MTBC lineages. This discovery further supports an East African origin for the MTBC and provides additional molecular clues on the ancestral genome reduction associated with adaptation to a pathogenic lifestyle.

### About the speaker
Conor Meehan is a lecturer in molecular microbiology at the University of Bradford. His research focusses on the evolution and epidemiology of microbes, primarily pathogenic mycobacteria such as *M. tuberculosis, M. ulcerans* and *M. leprae*, with an emphasis on whole genome sequencing coupled with phylogenetics-based analyses

https://www.bradford.ac.uk/staff/cmeehan2

Video: https://youtu.be/5x8B4J6hFFU

## From phylogenetics to phylogenomics: between bacterial strains and the tree of life
With the amount and velocity of genomic data currently available, it became clear that no single gene, or even carefully selected gene sets, can provide a single evolutionary tree with confidence. We can therefore appreciate that apparently conflicting phylogenetic signals are not noise, but in fact reflect real biological phenomena ―  like duplications, lateral transfer, or ancestral polymorphisms. Phylogenomic models explicitly recognise this variation in evolutionary histories across the genome, allowing us to make most use of sequenced genomes. In this talk we will have an overview of popular models, as well as of emerging methodologies that can handle whole genomes from hundreds or thousands of samples.  We will also discuss favourite approaches in different fields, where the genomic sets may resemble closer a “tall” (fewer information from many samples) or a “wide” (fewer samples, with more information) data category.

### About the speaker
I am the Head of Phylogenomics of the Quadram Institute Bioscience, where I’ve been working for one year supporting and creating analysis of large data sets. Before that, I’ve held postdoctoral positions at the University of Lausanne/Swiss Institute of Bioinformatics, at the Imperial College London, and at the University of Vigo in Spain. I worked mainly with the development of phylogenetic models and tools at the genome scale, usually focused at the tree-of-life level. I did my PhD at the University of Tokyo, Japan, designing a Bayesian phylogenetic method for the inference of viral recombination.   

https://leomrtns.github.io

Video: https://youtu.be/N0fkRuhr0RE

## Combating shigellosis in endemic settings with pathogen genomic 
The Global Enteric Multicentre Study (GEMS) identified *Shigella* as one of the leading causes of moderate-to-severe diarrhoea in children across seven African and Asian countries. However, increasing antimicrobial resistance (AMR) and lack of a licensed vaccine, means that treatment and management options for shigellosis are limited. Whole-genome sequencing analysis (WGSA) can be effectively used to investigate AMR and epidemiological surveillance of Shigella, but this has yet to be applied in countries where childhood mortality rates are the highest and the disease is endemic. Here, we performed WGSA of Shigella isolates from GEMS (255 *S. sonnei* and 759 *S. flexneri*) to explore the implications of AMR and genetic diversity relating to vaccination, to improve interventions and management of shigellosis.

We detected multiple, independent acquisitions of mutations conferring reduced susceptibility to ciprofloxacin (the current recommended treatment) among isolates from Asia, reflecting selection due to heavy usage of the drug in these regions. While vaccination is a good alternative for management of shigellosis, we revealed implications which may impact the likely effectiveness of multiple approaches. These include, sequence variation within known epitopes of several antigen-vaccine candidates (eg. IpaB). Serotype switching, which may compromise immune protection induced by serotype-specific glycoconjugate vaccines. To help inform effective implementation of such vaccination programs, temporal analysis was used to estimate anticipated time frames over which serotype switching may lead to vaccine escape. In conclusion, WGSA of this representative dataset has provided valuable information to guide treatment and management of shigellosis through antimicrobial therapy and vaccination.

## About the speaker
Rebecca is a postdoctoral research associate working for Dr Kate Baker at the department of functional and comparative genomics in the University of Liverpool, UK. She is currently working computationally on a large dataset of whole genome sequences from *Shigella* collected from Asia and Africa. Rebecca’s main research interests focuses on applying various statistical methods to investigate the molecular evolution, phylogenetics and epidemiology of Shigella. Her aim is to gain understanding of the processes that are driving disease emergence and spread.

https://baker-lab.github.io/

Video: https://youtu.be/M6qjwWG-u0o
