---
title: Session 1 (15th April 2020)
date: "2020-04-15T15:00:00Z"
layout: post
draft: false
path: "/posts/session-1/"
category: "Session Schedule"
tags:
  - "Population genetics and Microbial epidemiology"
  - "Bioinformatics software and protocols"
description: "With talks from Zhemin Zhou, Dan Fornika and Dag Harmsen"
---
# Schedule 

Time: 16:00 15 April 2020 (UTC) 

Zoom link: Coming soon.

Discord link: https://discord.gg/WmuRbD

* 15:00 **EnteroBase: exploit core genomic diversity of bacterial pathogens** 
  Zhemin Zhou (University of Warwick)  
* 15:30 **Intro to IRIDA Pipeline Plugins**  
  Dan Fornika (BCCDC Public Health Laboratory)
* 16:00 **cgMLST Re-visited: Evolvement and Challenges**  
  Dag Harmsen (Univ. M체nster, Germany)   
* 16:30 **Virtual coffee break**

## EnteroBase: exploit core genomic diversity of bacterial pathogens
EnteroBase is an integrated software environment which supports the identification of global population structures within several bacterial genera. EnteroBase has currently assembled more than 400,000 genomes from Illumina short reads from Salmonella, Escherichia and five other bacterial genera, and genotyped most of these genomes by core genome Multilocus Sequence Typing (cgMLST). Hierarchical clustering of cgMLST sequence types allows mapping a new bacterial strain to predefined population structures at multiple levels of resolution and thus dynamic tracking of transmission chains. The interactive graphical user interface in EnteroBase eases the access of large scale bacterial genomic data and facilitates international data sharing and collaborations. I demonstrated the use of EnteroBase with multiple examples in Salmonella and Escherichia

https://github.com/zheminzhou

## Intro to IRIDA pipeline plugins
Quick intro to the IRIDA system (irida.ca), followed by brief discussion of how to develop your own 'Pipeline Plugins'

Dan Fornika is a Molecular Biologist + Bioinformatician = 'Genomics Specialist' (https://github.com/dfornika)

## cgMLST re-visited: Evolvement and challenges - About the speaker
Dr. Dag Harmsen is an MD specialized in Clinical Microbiology and Infectious Diseases Epidemiology. Since 2004 he holds a professorship at the Medical Faculty of the University M체nster, Germany. His scientific interests focus around molecular diagnostics, epidemiology, and phylogeny of micro-organisms. He published the world-wide first study that applied the Bruker MALDI ToF MS system in microbiological diagnostics (JCM 2008; 46:1946). He earned quite some recognition for early real-time application of benchtop NGS sequencing and first application of what later become known as core genome MLST (cgMLST) genome-wide allele calling during the 2011 German STEC O104:H4 outbreak (PLoS ONE 2011; 6:e22751). More recently he published a highly noted NGS benchtop machine comparison (Nat Biotechnol 2013; 31:294). However, he is best known for delivering for more than 20 years popular applied microbiology bioinformatics solutions - like spa-typing for MRSA (JCM 2003; 41:5442) or the MIRU-VNTR service for M. tuberculosis typing (Nucleic Acids Res 2010; 38:W326).

Dr. Harmsen is also co-founder (2003), shareholder, and co-managing director of Ridom GmbH (M체nster, Germany). Ridom develops bioinformatics software (most notable StaphType and SeqSphere+) for clinical and public health microbiology sequence (S채nger and WGS) analysis. From 2005 until 2014 he was a member of the Executive Board of the 'International Committee on Systematics of Prokaryotes' (ICSP) of the International Union of Microbiological Societies (IUMS). From 2007 until July 2012 he served in the 'Professional Development Committee' of the American Society for Microbiology (ASM). Finally, Dr. Harmsen was technical advisor for NGS based surveillance of ECDC, EFSA, WHO, FAO, US Government Accountability Office, and US National Sciences Academy.